src/pybind/mgr/mgr_module.py

```diff
@@ -1027,7 +1021,12 @@ def get_store(self, key, default=None):
             return r
 
     def get_localized_store(self, key, default=None):
-        return self._get_localized(key, default, self.get_store)
+        r = self._ceph_get_store('{}/{}'.format(self.get_mgr_id(), key))
```

Comment:
Nothing really important but, could it make sense to encapsulate the code to generate the localized option name (either `{}/{}'format(id, key)` or `id + '/' + key`). I mention this because in the code it's done in the both ways.